integrations/launchdarkly/main.py

```diff
@@ -0,0 +1,97 @@
+from loguru import logger
+from port_ocean.context.ocean import ocean
+from client import LaunchDarklyClient
+from port_ocean.core.ocean_types import ASYNC_GENERATOR_RESYNC_TYPE
+from enum import StrEnum
+from typing import Any
+
+
+class ObjectKind(StrEnum):
+    PROJECT = "project"
+    AUDITLOG = "auditlog"
+
+class ResourceKindsWithSpecialHandling(StrEnum):
+    FEATURE_FLAGS = "flags"
+    ENVIRONMENTS ="environments"
```

Comment:
Change to singular (flag and environment) so it's consistent with the kinds you have defined in the spec.yaml

Reply:
Hmm, the plural naming, was purposed to easily sync with the launchdarkly endpoints .eg https://app.launchdarkly.com/api/v2/flags/{projectKey},
renaming these means code-wise, i can't rely on the kinds to perform the switches..

Reply:
But thats okay, I'll perform some workaround on that.